frontend/src/pages/TutorialPage/index.tsx

```diff
@@ -0,0 +1,187 @@
+import React from "react";
+import "./index.css";
+import { Helmet } from "react-helmet";
+import { useGoogleAnalytics } from "../../components/useGoogleAnalytics";
+import { Divider, Typography, Image } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+
+const { Title, Paragraph, Text, Link } = Typography;
+/**
+ * Tutorial Page
+ */
+
+export function TutorialPage() {
+  useGoogleAnalytics();
+
+  return (
+    <div className="page">
+      <Helmet>
+        <title>ExplainaBoard - Tutorials</title>
+      </Helmet>
+      <Typography>
+        <Title>What is ExplainaBoard? Why Do We Need It?</Title>
+        <Paragraph>
+          The reason why we created ExplainaBoard is to do automatic evaluation
+          for users. We want to be able to save time after they already took a
+          long time training and tuning their models.
+        </Paragraph>
+
+        <Title level={2}>Getting Started</Title>
+        <Paragraph>
+          In this quickstart guide, we will walk you through the basic
+          functionalities of ExplainaBoard:
+        </Paragraph>
+        <Paragraph>
+          <ul>
+            <li>
+              <a href="#upload-system">Upload a system </a>
+            </li>
+            <li>
+              <a href="#view-analysis">
+                View detailed analysis of your outputs
+              </a>
+            </li>
+          </ul>
+        </Paragraph>
+        <Divider />
+        <Title level={2} id="upload-system">
+          Upload a System
+        </Title>
+        <Paragraph>
+          We will be using a text classification example for this tutorial.
+          Please download the <Text code>output_sst2.txt</Text> file{" "}
+          <Link href="https://github.com/neulab/ExplainaBoard/blob/main/integration_tests/artifacts/text_classification/output_sst2.txt">
+            here
+          </Link>{" "}
+          which contains a model prediction for the sst2 dataset.
+          <ol>
+            <li>
+              On the side menu on the right, click on the{" "}
+              <Text strong>Systems</Text> tab.
+            </li>
+            <Image
+              src={"/screenshots/systemsTab.png"}
+              preview={false}
+              style={{
+                width: "200px",
+                maxWidth: "90%",
+                margin: "10px 0 10px 100px",
+              }}
+            />
+            <li>
+              Along the filter bar in the systems page, click the{" "}
+              <Text strong>New</Text> button.
```

Comment:
These directions and screenshot are out of date. The most recent interface has a green "Submit New System" button.